apps/cli/src/helpers/setup/frontend-setup.ts

```
@@ -0,0 +1,38 @@
+import consola from "consola";
+import pc from "picocolors";
+import type { ProjectConfig } from "../../types";
+import { addPackageDependency } from "../../utils/add-package-deps";
+
+export async function setupFrontendDependencies(config: ProjectConfig) {
+	const { frontend, projectDir } = config;
+
+	// Check if nativewind is being used
+	const hasNativeWind = frontend.includes("native-nativewind");
+	const hasUnistyles = frontend.includes("native-unistyles");
```

Biome (2.1.2)

[error] This variable hasUnistyles is unused.

Unused variables are often the result of an incomplete refactoring, typos, or other sources of bugs.
Unsafe fix: If this is intentional, prepend hasUnistyles with an underscore.


(lint/correctness/noUnusedVariables)